### DISCUSSION TITLE
Hey I am unable to delete my Vercel team

### DISCUSSION THREAD

**Discussion by supratikmahato** [vercel/community#71] [Help] [answered]
I was on team pro trial a few weeks ago. So when I opened the team dashboard today, I saw that's its overdue and it won't let me delete the team. What should I do next?

**Comment by ismaelrumzan** [comment, marked as answer]
Hi there.
The trial period is 14 days. After that, your team account switches to a team pro account. Are you looking to cancel the team account and only use your personal account?

**Reply by supratikmahato** [reply]
Hey yeah I want to do that
